Dockerfile

```diff
@@ -1,5 +1,12 @@
-FROM golang:1.7
-ADD . /go/src/go.uber.org/yarpc
-RUN go install go.uber.org/yarpc/internal/crossdock
-ENTRYPOINT /go/bin/crossdock
+FROM golang:1.7.4
+
 EXPOSE 8080-8087
+RUN mkdir -p /go/src/go.uber.org/yarpc
+WORKDIR /go/src/go.uber.org/yarpc
+ADD glide.yaml /go/src/go.uber.org/yarpc/
+ADD glide.lock /go/src/go.uber.org/yarpc/
```

Comment:
So I actually did this on purpose, but in practice it's silly - glide.yaml is less likely to change than glide.lock, so caching patterns etc..but I think it's even slower. Also for full disclosure, I've been working with docker for two years and I never knew you could do multiple files on the same ADD line *embarrassing*. Changing.